src/unity/python/turicreate/toolkits/drawing_classifier/drawing_classifier.py

```diff
@@ -51,9 +51,9 @@ def create(input_dataset, feature="drawing", target="label",
     start_time = _time.time()
 
     if max_iterations == 0:
-        max_iterations = num_epochs * len(input_dataset) / batch_size
-    else:
-        num_epochs = max_iterations * batch_size / len(input_dataset)
+        # @TODO: Be able to automatically choose number of epochs based on 
+        # data size
+        max_iterations = 100
```

Comment:
Why not just set the default in line 42?

Reply:
Fixed.